conans/client/generators/cmake_common.py

```diff
@@ -850,7 +850,7 @@ class CMakeCommonMacros:
                 endif()
                 foreach(_FRAMEWORK ${FRAMEWORKS})
                     # https://cmake.org/pipermail/cmake-developers/2017-August/030199.html
-                    find_library(CONAN_FRAMEWORK_${_FRAMEWORK}_FOUND NAME ${_FRAMEWORK} PATHS ${CONAN_FRAMEWORK_DIRS${SUFFIX}})
+                    find_library(CONAN_FRAMEWORK_${_FRAMEWORK}_FOUND NAME ${_FRAMEWORK} PATHS ${CONAN_FRAMEWORK_DIRS${SUFFIX}} CMAKE_FIND_ROOT_PATH_BOTH)
```

Comment:
It would be great to know when we could use ``NO_DEFAULT_PATH``, because we fully know this is a framework that lives in the Conan package, and forget about all the system paths, and when it would be a system framework and then it could be found in other locations rather than in the ``${CONAN_FRAMEWORK_DIRS${SUFFIX}} ``

Now that we have the ``system_libs``, this could be the same for frameworks? ``system_frameworks``?

Wdyt @danimtb, @jgsogo ?

Reply:
Makes sense. It would be a better model.

Reply:
Agree. Then we should turn this bugfix into a feature for 1.29

Reply:
yes, I think having a `cpp_info.system_frameworks` makes sense

Reply:
it makes sense, but it will be much larger change - e.g. update several generators at least

Reply:
yes, I am not saying that it is a change for this PR, but something to consider for a feature